packages/resources/test/Api.test.ts

```diff
@@ -208,7 +210,7 @@ test("accessLog-string", async () => {
   );
 });
 
-test("accessLog-props", async () => {
+test("accessLog-settings", async () => {
```

Comment:
"accessLog-format" might make more sense for the name.

Reply:
Yes it does. Updated.